src/core/coreconfig.cpp

```diff
@@ -49,6 +53,8 @@ QJsonObject CoreConfig::toJson() const
     obj[QStringLiteral("locale")] = m_locale;
     obj[QStringLiteral("shortcuts")] = saveShortcuts();
     obj[QStringLiteral("toolbar_icon_size")] = m_toolBarIconSize;
+    if(m_minimize_to_system_tray!=-1)
```

Comment:
If we use `int`, we could always store the value without condition check.

Reply:
Yes, I did this before because I didn’t want to modify the default config template, so add function isUnDefinedKey

Reply:
When saving, I want to save true or false for beauty